static/css/custom.css

```diff
@@ -0,0 +1,3 @@
+.tab:is(input[type="radio"]) {
```

Comment:
Nice. We use the [site-tailwind.css](https://github.com/dimagi/open-chat-studio/blob/main/assets/styles/site-tailwind.css) file for custom styles (there are a few other files as well, but this seems to be the correct place to put this style override). Once it's in there, you can run `npm run dev`/`npm run build` for it to take affect. (I recently started running `npm run dev-watch` while developing. This way new styles are always being "pulled in")

Reply:
ahh good info. I went with the site-tailwind.css change [here](https://github.com/dimagi/open-chat-studio/pull/423#:~:text=width%20for%20tabs-,1275a62,-remove%20redundant%20subtitles)

Reply:
Looks like you manually added the site-tailwind.css file. So we don't have to do that. There is [a file](https://github.com/dimagi/open-chat-studio/blob/main/assets/styles/site-tailwind.css) called site-tailwind.css in the assets/styles directory in which you can put the new style and run any one of these commands to load/apply the style: `npm run build`/`npm run dev`/`npm run dev-watch`. This will pull all JS and css/scss files from the assets directory, package them and put them in the static/ directory where the server can find them (see the webpack.config.js file. Not sure how well versed you are with webpack, but [here's](https://webpack.js.org/concepts/configuration/) their explanation of what's going on in there).

The static/js and static/css directories are being built each time we deploy, so we don't have to worry about putting anything in there manually. I hope I made sense :sweat_smile:

Reply:
@SmittieC ah gotcha thanks for the explanation (that makes a lot of more sense)! I added change here and ran the command locally and got the expected results but lmk if I'm still not understanding it fully: https://github.com/dimagi/open-chat-studio/pull/423/commits/509fe0500685ae8ba0b5ae66a4317f6c47599eea